metastore/scripts/upgrade/hive/hive-schema-4.0.0-beta-2.hive.sql

```diff
@@ -719,8 +719,6 @@ FROM
 
 CREATE EXTERNAL TABLE IF NOT EXISTS `TAB_COL_STATS` (
  `CS_ID` bigint,
- `DB_NAME` string,
```

Comment:
those are system views, not sure we should drop cols from here

Reply:
The `.hive.sql` seems only be used in test, we can join the `DBS` if these cols are necessary.

Reply:
sure